imp/NonTerminals.cs

```diff
@@ -57,10 +57,304 @@ public class NonTerminals : INonTerminals {
                              lastNoVariantRecDependent = Production.TypeDeclarationTail,
                              firstOptionDependent = firstConstParamDependent,
                              lastOptionDependent = lastNoVariantRecDependent;
-    public static uint alternateSetOffset = (uint)lastOptionDependent - (uint)firstOptionDependent + 1;               
-    
-    TokenSet[] followSet = new TokenSet[Count()];
-    TokenSet[] firstSet = new TokenSet[Count()];
+    public static uint alternateSetOffset = (uint)lastOptionDependent - (uint)firstOptionDependent + 1;

```

Comment:
The constants firstConstParamDependent, lastConstParamDependent, firstNoVariantRecDependent, lastNoVariantRecDependent, firstOptionDependent, lastOptionDependent and alternateSetOffset are only used by the methods of this class, we don't need to expose them to clients. They ought to be private.